src/scripts/ci_build.py

```diff
@@ -178,7 +185,7 @@ def determine_flags(target, target_os, target_cpu, target_cc, cc_bin,
     if target == 'fuzzers':
         flags += ['--unsafe-fuzzer-mode', '--terminate-on-asserts']
 
-    if target in ['fuzzers', 'coverage']:
+    if target in ['fuzzers', 'coverage', 'sanitizer']:
         flags += ['--build-fuzzers=test']
```

Comment:
With the clarifications of the differences between the fuzzers and sanitizer jobs is does not seem like we want to build the fuzzer tests also for the sanitizer jobs.

Reply:
That makes sense to me, too. Changed.